index.html

```diff
@@ -42,177 +44,169 @@ <h1>Get ready to Spot the Surveillance...</h1>
-  </a-mixin>
-  <a-mixin id="arrow"  geometry="primitive: plane; width: 2; height: 2; " material="shader: flat; side: front; opacity: 1; transparent: true; src:#right-arrow"  position="0 -5 -2" visible="false" >
-  </a-mixin>
-
-  <!-- misc mixins -->
-  <a-mixin id="see-through" material="color: white; transparent: true; opacity: 0;"  scale="3 3 1">
-  </a-mixin>
-  <!-- this is the initial position of all hotspots/devices, so they are hidden behind the exit until the scene loads in load-scene.js-->
-  <a-mixin id="spotted"  geometry="primitive: plane; width: 5; height: 5; " material="shader: flat; side: front;  transparent: true; src:#found;" position="47.468 -341 10.223">
-  </a-mixin>
-  <a-mixin id="counter"  geometry="primitive: plane; width: 1.5; height: 0.5; " material="shader: flat; side: front;  transparent: true; src:#counter-bg;" text="align: center; color: white; width: 2; height: 1 ; xOffset: 0.15; value: 0 / 7" position="0 -0.85 -2.2">
-  </a-mixin>
-  <a-mixin id="close"  event-set__enter="_event: mouseenter; _target: #alpr; visible: false"
-  position="1.6 1.7 1.2" geometry="primitive: plane; width: 1; height: 1;" material="color: black; shader: flat; side: front; opacity: 0; transparent: true; visible: false;" >
-</a-mixin>
-
-<a-mixin id="exit-logo" event-set__enter="_event: mouseenter; _target: #exit-popup; visible: true"  geometry="primitive: plane; width: 50; height: 15;" material="transparent: true; opacity: 1; src:#eff-logo" position="8 -40.524 2.747" rotation="-90 0 0" visible="true" >
-</a-mixin>
-<a-mixin id="exit" geometry="primitive: sphere"  material="color: #ec1e1e" position="47.468 -340.523 10.223"  scale="300 1 300">
-</a-mixin>
-</a-assets>
-
-
-<a-entity camera look-controls wasd-controls position="0 2.2 4" >
-  <a-entity id="cursor" cursor="maxDistance: 30; fuse: true"
-  geometry="primitive: ring; radiusInner: 0.1; radiusOuter: 0.15;"
-  material="color: white; shader: flat;"  position="0 0 -3"
-   raycaster="objects: [data-clickable]";>
-  <a-animation begin="click" easing="ease-in" attribute="scale"
-  fill="forwards" from="0.2 0.2 0.2" to="1 1 1" dur="150"></a-animation>
-  <a-animation begin="fusing" easing="ease-in" attribute="scale"
-  fill="backwards" from="1 1 1" to="0.2 0.2 0.2" dur="1500"></a-animation>
-</a-entity>
-<a-entity mixin="counter" id="count-display">
-</a-entity>
-</a-entity>
-
-<!-- intro -->
-<a-entity  id="intro-popup" mixin="intro" sound="loop: false;">
-  <a-entity id="get-started" loadscene mixin="close" data-clickable  event-set__enter="_event: click; _target: #intro-popup; visible: false" geometry="primitive: plane; height: 1; width: 3" material="visible: false; transparent: true; opacity: 0;" position="0.072 -1.707 0.905" rotation="0 -20 0">
-  </a-entity>
-</a-entity>
+        <audio id="ambience" src="assets/audio/traffic-fade.mp3" preload="auto"></audio>
```

Comment:
_(apologies if I'm getting confused with your other PRs.)_

I thought you determined that loading the sounds as `<audio>` tags in `<a-assets>` was affecting the perf?

[here's the relevant `<a-assets>` code](https://github.com/aframevr/aframe/blob/2a899cf/src/core/a-assets.js#L51-L59) in the A-Frame source for reference.

<hr>

on a related note: how does the loading work with `preload="auto"`?

I think it's Firefox, right, that requires `<audio preload="auto" autoplay>` to do proper autoplay, correct?

the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Apps/Fundamentals/Audio_and_video_delivery/Cross-browser_audio_basics#preload) suggest that the browser behaviour varies, especially on mobile.

lol, sounds like a good Hacks blog post to write about to inform web developers of best practices and gotchas with media playback, especially in VR.

Reply:
I'm leaving the `preload="auto"` for desktop, these fixes are only for FXR or Go where we have issues with preloading causing stutter.  It works for everything else.

Reply:
will also fix in another PR.